features/environment.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=W0601, W0613
+from behave import use_fixture
+
+from features.fixtures import selenium_browser
+
+
+def setup_debug_on_error(userdata):
+    global BEHAVE_DEBUG_ON_ERROR
```

Comment:
Why do you need this to be `global`?

Reply:
The code was taken directly from behave docs, as a proposed way to handle debugging by devs. https://behave.readthedocs.io/en/latest/tutorial.html#debug-on-error-in-case-of-step-failures

Reply:
My bad, I forgot I got rid of this functionality, since I figured it wasn't really useful for me to call debugger after the fact. So this is a part of the code that I forgot to remove.

Unless you guys think this could be useful and I should bring this functionality back?

Reply:
I removed it for now.

Reply:
Lets don't use globals if not needed (which is - never ;) )